Win/Mineways.cpp

```diff
@@ -1478,6 +1499,24 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
                 }
             }
             break;
+        case IDM_FILE_PUBLISHSKFB:
+            // Force it to be an rendering export: Relative obj
+            gPrintModel = 0;
+            gExportViewData.fileType = FILE_TYPE_WAVEFRONT_REL_OBJ;
```

Comment:
It is not consistent in the whole file ([here](https://github.com/sketchfab/Mineways/pull/2/files#diff-5ff29cdf6af1546951a5781ddf0e05f4L1403) or [here](https://github.com/sketchfab/Mineways/pull/2/files#diff-5ff29cdf6af1546951a5781ddf0e05f4R1437) ) and it is more readable with spaces so I don't know which one to use. But I could be consistent in my own code